docs/.vitepress/data/blog.data.js

```diff
@@ -36,7 +40,8 @@ export default {
 
 		return {
 			blog: {
-				articles: posts,
+				articles: published,
+				hidden: hidden,
```

Comment:
Do we need to expose (or even fetch) hidden articles? Right now it doesn't seem like it's used anywhere.

Reply:
Yes good point. Not from here. They are rendered as they aren't excluded in  `docs/blog/[slug].paths.js`. We can actually nip out the declaration and returning of `hidden`.

Reply:
Perfect 👍 Updated now!